chalk-parse/src/ast.rs

```diff
@@ -177,6 +177,11 @@ impl PolarizedTraitRef {
     }
 }
 
+pub struct TraitBound {
```

Comment:
I think it would be useful to add a comment here explaining what this represents and how it is used.

Reply:
done!